packages/block-library/src/navigation-menu-item/editor.scss

```diff
@@ -66,8 +66,7 @@
 .wp-block-navigation-menu-item {
 	&:not(.is-editing) {
 		font-family: inherit;
-		font-size: $text-editor-font-size;
-		font-weight: bold;
+		font-size: inherit;
```

Comment:
Can the font-family & font-size rules be removed completely? They didn't seem to have any effect when I disabled them in dev tools. I think `inherit` is the default.